src/test/groovy/org/prebid/server/functional/model/request/auction/Dooh.groovy

```diff
@@ -0,0 +1,16 @@
+package org.prebid.server.functional.model.request.auction
+
+import groovy.transform.ToString
+
+@ToString(includeNames = true, ignoreNulls = true)
+class Dooh {
+    
+    String id
+    String name
+    List<String> venuetype
+    Integer venuetypetax
```

Comment:
Why are we using lowercase instead of `camelCase` and @JsonProperty?

Reply:
+1
Same for `venuetype`